Lesson27/feyza/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Posts</title>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <h1>Posts</h1>
+      <button id="fetch-button" class="button button--success">
+        Fetch posts
+      </button>
+
+      <a href="../feyza/create/create-post.html">
```

Comment:
Url should be `"./create/create-post.html"`. '.' means the current folder.

When you use '..' you go one level up (so in this case you go to Lesson27 folder, then come back to the 'feyza' folder. This is unnecessary.